.gitignore

```diff
@@ -275,3 +275,7 @@ GitHub.sublime-settings
 
 *hmr.js
 .vscode/settings.json
+
+dash_vtk
```

Comment:
Why are we removing this?

Reply:
I'm not removing it per say except that an new file in it won't be detected as "should be tracked"

Reply:
The `dash_vtk.min.js` files are used by `Dash` when you import `dash_vtk`. So if it changes then the changes will need to be reflected or someone that's installing the library with `pip install git+https://github.com/plotly/dash-vtk.git` will get an error since the `dash_vtk.min.js` is out of date.